core/dbt/parser/manifest.py

```diff
@@ -808,8 +808,12 @@ def inject_external_nodes(self) -> bool:
         plugin_model_nodes = pm.get_nodes().models
         for node_arg in plugin_model_nodes.values():
             node = ModelNode.from_args(node_arg)
-            # node may already exist from package or running project - in which case we should avoid clobbering it with an external node
-            if node.unique_id not in self.manifest.nodes:
+            # node may already exist from package or running project (even if it is disabled),
+            # in which case we should avoid clobbering it with an external node
+            if (
+                node.unique_id not in self.manifest.nodes
```

Comment:
I think this could end up being quite noisy as a warning because it would likely be multiple project nodes (e.g. from a package) being favoured over a collection of external nodes. I've been viewing it as something similar to our other precedence / resolution behaviour for resolving macros, refs, etc which is just 'expected' behaviour. Definitely warrants documentation though!

Reply:
Agree, I don't think we want a warning here. We will document [here](https://docs.getdbt.com/docs/collaborate/govern/project-dependencies#comparison) that, in _all_ cases where identical resources from the same namespace are installed via both "project" and "package" dependency, the "package" dependency is preferred, whether or not the resources from that package are actually `enabled`.

Reply:
Docs issue:
- https://github.com/dbt-labs/docs.getdbt.com/issues/5571